Automattic/wp-calypso
# Playwright: migrate Auth Screen canary test as demonstration

#### Changes proposed in this Pull Request

Migrate the Auth Screen canary test case from `wp-log-in-out-spec.js` file

The migration in this PR is very basic. None of the driver helpers, driver managers and other required tooling have been migrated; those will come in subsequent steps. 

The purpose of this PR is to try out the basic canary test with minimal scaffolding to verify that it can run on TeamCity as a regular task, and that basic foundation elements are in place for a super basic Playwright test to run.

In this PR, the decision has been made to:
- store migrated tests under a separate directory `specs-playwright`.
- create a new step in TeamCity, to be run after the Selenium tests have run.

In a future PR, scaffolding code such as playwright-specific `before/after` hooks, driver managers and other utilities will be implemented.

#### Testing instructions

Ensure the [build](https://teamcity.a8c.com/buildConfiguration/calypso_RunCalypsoPlaywrightE2eTests/6015452) completes without any errors.

-or-

1. check out the branch
2. `yarn` to install dependencies
3. run either `yarn magellan --config=magellan-playwright.json --max_workers=%E2E_WORKERS% --suiteTag=parallel --local_browser=chrome --mocha_args="--reporter mocha-teamcity-reporter" --test=specs-playwright/wp-log-in-out-spec.js` or `mocha specs-playwright`

Implements #51080
